docs/source/integrations/triton.rst

```diff
@@ -1,3 +1,599 @@
-======
-Triton
-======
\ No newline at end of file
+=======================
+Triton Inference Server
+=======================
+
+*time expected: 12 minutes*
+
+NVIDIA Triton Inference Server [#triton]_ is a high performance, open-source inference server for serving deep learning models.
+It is optimized to deploy models from multiple deep learning frameworks, including TensorRT,
+TensorFlow, ONNX, to various deployments target and cloud providers. Triton is also designed with 
+optimizations to maximize hardware utilization through concurrent model execution and efficient batching strategies.
+
+BentoML now supports running Triton Inference Server through the :ref:`Runner <concepts/runner:Using Runners>`
+architecture. The following integration guide makes the assumption that readers are familiar with BentoML infrastructure.
+Make sure to check out our :ref:`tutorial <tutorial:Creating a Service>` should you wish to learn more about BentoML.
+
+The guide will try to be as comprehensive and detailed as possible, yet we will not be able to cover all the
+features from Triton Inference Server. For more information, please refer to their documentation [#triton_docs]_.
+
+The code examples in this guide can also be found in our examples folder [#triton_runner]_.
+
+
+Prerequisites
+~~~~~~~~~~~~~
+
+Make sure to have at least BentoML 1.0.14 and ``tritonclient`` at least version 2.29.0 available in your Python environment:
+
+.. code-block:: bash
+
+    $ pip install -U bentoml tritonclient[all]
+
+.. note::
+
+   Triton Inference Server is currently only available in production mode (``--production`` flag) and will not work during development mode.
+
+Additonally, you will need to have Triton Inference Server installed in your system. Refer to their building documentation [#triton_build]_
+to setup your environment.
+
+The recommended way to run Triton is through container (Docker/Podman). To pull the latest Triton container for testing, run:
+
+.. code-block:: bash
+
+    $ docker pull nvcr.io/nvidia/tritonserver:<yy>.<mm>-py3
+
+.. note::
+
+    ``<yy>.<mm>``: the version of Triton you wish to use. Currently the latest version is ``22.12``.
+
+In this guide, we will demonstrate the capabilities of Triton Inference Server with BentoML and provide constructive
```

Comment:
What do you mean by constructive recommendation?

Reply:
I think i just meant recommendation